packages/next/src/server/app-render/use-flight-response.tsx

```diff
@@ -158,27 +166,25 @@ export function createInlinedDataReadableStream(
 function writeInitialInstructions(
   controller: ReadableStreamDefaultController,
   scriptStart: string,
-  formState: unknown | null
+  formState: unknown | null,
+  requestId: string
 ) {
+  let scriptContents = `(self.__next_f=self.__next_f||[]).push(${htmlEscapeJsonString(
+    JSON.stringify([INLINE_FLIGHT_PAYLOAD_BOOTSTRAP])
+  )})`
+
+  if (process.env.NODE_ENV !== 'production') {
+    // The request ID is only needed in development mode.
+    scriptContents = `self.__next_r=${JSON.stringify(requestId)};${scriptContents}`
```

Comment:
should this be preceded by a semicolon like we do in the `formState != null` branch below?